jmix-search/search/src/main/java/io/jmix/search/index/fileparsing/FileParserResolver.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.search.index.fileparsing;
+
+import io.jmix.core.FileRef;
+import org.apache.tika.parser.Parser;
+
+import java.util.List;
+
+/**
+ * Is a part of the extendable engine the gives an ability to implement custom file parser resolvers and to support
+ * custom file types or to modify behavior of existing file parser resolvers.
+ */
+public interface FileParserResolver {
+
+    /**
+     * This method should return the description that describes the constraints or the constraint for the files
```

Comment:
JavaDoc is written for API users, not for developers who will implement it. Writing style for a methods: starting with the verb defining the main action.

Check https://youtrack.haulmont.com/articles/PLAT-A-1158/Code-Style#Javadocs

Reply:
Fixed